lib/src/main/java/com/auth0/jwt/JWTVerifier.java

```diff
@@ -356,6 +356,23 @@ public DecodedJWT verify(String token) throws JWTVerificationException {
         return jwt;
```

Comment:
DRY: The existing method should delegate to the new method with the decoded token.

Reply:
done